kuma/users/management/commands/configure_social_auth.py

```diff
@@ -0,0 +1,98 @@
+import fileinput
+import os
+import sys
+
+from allauth.socialaccount.models import SocialApp
+from django.conf import settings
+from django.contrib.sites.models import Site
+from django.core.management.base import BaseCommand
+
+LOCALHOST = 'localhost.org:8000'
+DOMAIN = 'localhost.org'
+
+OVERWRITE_PROMPT = 'There\'s already a SocialApp for %s, if you want to overwrite it type "yes":'
+GITHUB_INFO = (
+    'Visit https://github.com/settings/developers and click "New OAuth App"\n'
+    'Set "Homepage URL" to "http://localhost.org:8000/" and Authorization callback URL to ' +
+    '"http://localhost.org:8000/users/github/login/callback/" respectively'
+)
+GOOGLE_INFO = (
+    'Visit https://console.developers.google.com/projectcreate and create a new project\n'
+    'After that visit https://console.developers.google.com/apis/credentials and add the following URLs:\n'
+    'Authorized JavaScript origins: http://localhost.org:8000\n'
+    'Authorized redirect URIs: http://localhost.org:8000/users/google/login/callback/\n'
+)
+ENV_INFO = 'Putting SITE_ID and DOMAIN into .env'
+HOSTS_INFO = (
+    'You should should run "docker-compose restart" now, so that the .env updates are applied\n.'
```

Comment:
I just tested it with `restart`, a print in `common.py` and all three python processes logged the new value.

Reply:
Hm, I'm surprised that `docker-compose restart` worked, as it was my understanding from past experience that a `docker-compose stop`+`docker-compose up` was required as well.

Reply:
I think I get it. It depends on which env vars it is. 
For example, I don't have `SENTRY_DSN` mentioned at all in `docker-compose.yml`. 
So, if I edit the `.env` file and then trigger a restart, Python (via `./manage.py') will start from scratch and re-read the `.env` file from disk. 
However, take `DEBUG` for example. If I mess with that in my `.env` it will never change until I do a `docker-compose stop ; docker-compose up`. 
That's because of how `docker-compose` works and `python-decouple` doesn't read this value from the file on disk, but from the environment. 
This is because we have this in `docker-compose.yml`:
```
DEBUG=${DEBUG:-True}
```

I genuinely believe we should go for https://github.com/mdn/kuma/pull/5390 instead of listing every possible env var in `docker-compose.yml`. 

Truth be told, I can't guarantee that that'll solve the problem because maybe what `docker-compose` does is that it reads the whole file when you `docker-compose up` and pushes it into memory and only `docker-compose stop` can clear that. 

Either way, the correct instruction is that you have to (as of today) run `docker-compose stop; docker-compose up` if you edit `DOMAIN` because that's listed in `docker-compose.yml`.

Reply:
Thanks for the deep-dive, Peter! That makes some sense (in the unfortunate sense of the word).

I've changed it to stop'n'up instead of restart.